README.md

```diff
@@ -60,6 +60,7 @@ Check out our test results:
 Up-to-date documentation for the MongooseIM master branch can be found on ReadTheDocs: [https://esl.github.io/MongooseDocs/latest/](https://esl.github.io/MongooseDocs/latest/).
 
 Latest releases:
+* [4.1.0](https://esl.github.io/MongooseDocs/4.1.0/)
```

Comment:
We need to check the docs as well to make sure they work.

Reply:
I think that's the last bullet of this task's description 😉